tests/basics/gc1.py

```diff
@@ -20,3 +20,10 @@
     # just test they execute and return an int
     assert type(gc.mem_free()) is int
     assert type(gc.mem_alloc()) is int
+
+if hasattr(gc, 'threshold'):
+    # uPy combines get/set_threshold function
```

Comment:
Well, it not just combines, it's pretty different function from CPython's, so better to use the same formulation as above "uPy has this extra function".